### DISCUSSION TITLE
Help on Learning Rate Scheduler with CombinedDataLoader

### DISCUSSION THREAD

**Discussion by leighgable** [Lightning-AI/pytorch-lightning#19747] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hello All,
I have data from a number of sites, which I have to keep seperate during training, so I am using the lightning combined dataloader. I would like to use a learning rate scheduler, but since the validation loss is prefixed by the dataloader index, 
I am not sure what to pass to `optim.lr_scheduler.ReduceLROnPlateau` for the `monitor` value. Can I hijack whatever callback is producing the validation loss with the path added to the name? I'm using the CSVLogger by the way. And the model is an LSTM/Autoencoder.

Any help would be greatly appreciated.
